utils/cliutils/commandsflags.go

```diff
@@ -1124,6 +1127,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  licenses,
 		Usage: "[Optional] Set to true if you'd like to receive licenses from Xray scanning. ` `",
 	},
+	vulnerabilities: cli.BoolFlag{
+		Name:  vulnerabilities,
+		Usage: "[Optional] Set to true if you'd like to receive vulnerabilities from Xray build scanning. ` `",
```

Comment:
[Optional] --> [Default: false]

Set to true if you'd like to receive vulnerabilities from Xray build scanning.
-->
Set to true if you'd like to receive all vulnerabilities, regardless of the policy configured in Xray